pkg/k8s/authz.go

```diff
@@ -109,7 +109,11 @@ func ServiceProfilesAccess(k8sClient kubernetes.Interface) error {
 // access to EndpointSlice resources.
 //TODO: Uncomment function and change return type once EndpointSlices
 // are supported and made opt-in through install flag
-func EndpointSliceAccess(k8sClient kubernetes.Interface) bool {
+func EndpointSliceAccess(k8sClient kubernetes.Interface, enableEndpointSlice bool) bool {
```

Comment:
let's leave the second parameter off of this method.  This should just check if we have access.